pages/profile.tsx

```diff
@@ -58,6 +61,10 @@ export default createPage({
   title: "Profile",
   Page: requireAuth(({ user: { displayName } }) => {
     const profile = useProfile()
+
+    const { user } = useAuth()
```

Comment:
You can get the uid from the user on line 62: 
`  Page: requireAuth(({ user: { displayName, uid } }) => {`